iron-multi-selectable.html

```diff
@@ -86,11 +86,16 @@
     },
 
     _selectMulti: function(values) {
-      this._selection.clear();
       if (values) {
-        for (var i = 0; i < values.length; i++) {
-          this._selection.setItemSelected(this._valueToItem(values[i]), true);
+        var selectedItems = this._valuesToItems(values);
+        // clear all but the current selected items
+        this._selection.clear(selectedItems);
+        // select only those not selected yet
```

Comment:
Actually, nevermind. The code below seems to plainly counter what the comment says unless you look into the implementation of `setItemSelected`; but, the point of a comment is to make things that aren't obvious from the implementation clear, right?